src/AuthUserService.php

```diff
@@ -63,8 +65,7 @@ public function getSession()
      */
     public function auth(string $requestHandler, ServerRequestInterface $request): bool
     {
-        echo sprintf(" 你访问了: %s",$requestHandler);
-        return true;
+        throw new AuthException(ErrorCode::POST_DATA_NOT_PROVIDED,"you need copy AuthUserService::auth method");
```

Comment:
throw new AuthException(ErrorCode::POST_DATA_NOT_PROVIDED, sprintf('AuthUserService::auth() method should be implemented in %s', static::class));